packages/date-picker/test/value-commit-auto-open-disabled.common.js

```diff
@@ -0,0 +1,255 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextRender, outsideClick } from '@vaadin/testing-helpers';
+import { sendKeys } from '@web/test-runner-commands';
+import sinon from 'sinon';
+
+const TODAY_DATE = new Date().toISOString().split('T')[0];
+
+describe('value commit - autoOpenDisabled', () => {
+  let datePicker, valueChangedSpy, validateSpy, changeSpy;
+
+  function expectNoValueCommit() {
+    expect(valueChangedSpy).to.be.not.called;
+    expect(validateSpy).to.be.not.called;
+    expect(changeSpy).to.be.not.called;
+  }
+
+  function expectValueCommit(value) {
+    expect(valueChangedSpy).to.be.calledOnce;
+    // TODO: Optimize the number of validation runs.
+    expect(validateSpy).to.be.called;
+    expect(validateSpy.firstCall).to.be.calledAfter(valueChangedSpy.firstCall);
+    expect(changeSpy).to.be.calledOnce;
+    expect(changeSpy.firstCall).to.be.calledAfter(validateSpy.firstCall);
+    expect(datePicker.value).to.equal(value);
+  }
+
+  function expectValidationOnly() {
+    expect(valueChangedSpy).to.be.not.called;
+    expect(validateSpy).to.be.calledOnce;
+    expect(changeSpy).to.be.not.called;
+  }
+
+  beforeEach(async () => {
+    datePicker = fixtureSync(`<vaadin-date-picker auto-open-disabled></vaadin-date-picker>`);
+    await nextRender();
+    validateSpy = sinon.spy(datePicker, 'validate').named('validateSpy');
+
+    valueChangedSpy = sinon.spy().named('valueChangedSpy');
+    datePicker.addEventListener('value-changed', valueChangedSpy);
+
+    changeSpy = sinon.spy().named('changeSpy');
+    datePicker.addEventListener('change', changeSpy);
+
+    datePicker.focus();
+  });
+
+  describe('default', () => {
+    it('should not commit but validate on blur', () => {
+      datePicker.blur();
+      expectValidationOnly();
+    });
+
+    it('should not commit but validate on Enter', async () => {
+      await sendKeys({ press: 'Enter' });
+      expectValidationOnly();
+    });
+
+    it('should not commit but validate on outside click', async () => {
+      outsideClick();
+      expectValidationOnly();
+    });
+
+    it('should not commit on Escape', async () => {
+      await sendKeys({ press: 'Escape' });
+      expectNoValueCommit();
+    });
+  });
+
+  describe('entering parsable input', () => {
+    beforeEach(async () => {
+      await sendKeys({ type: '1/1/2001' });
+    });
+
+    it('should commit on blur', async () => {
```

Comment:
nit: async is not needed here.

```suggestion
    it('should commit on blur', () => {
```